### PR TITLE
additional corrections to lumino doc

### PR DESCRIPTION
**Pull request by mortelli**
### Pending

- [x] resolve conflicts
- [x] do a self-review for this PR
- [x] try out the explorer API guide
- [x] try out the explorer WEB guide
- [ ] try out the lumino SDK js guide
- [x] finish reading the devportal `README.md` file to make sure we are following all guidelines

### CODE REVIEW

File: rif/lumino/explorer/index.md [hunk 1]
```diff
@@ -1,42 +1,40 @@
...
 ---
 
-Repository: [https://github.com/rsksmart/lumino-explorer](https://github.com/rsksmart/lumino-explorer)
 
-Mainnet public explorer: [https://explorer.lumino.rifos.org/](https://explorer.lumino.rifos.org/)
+- [Repository](https://github.com/rsksmart/lumino-explorer)
+- [Mainnet Public Explorer](https://explorer.lumino.rifos.org/)
 
-RIF Lumino Explorer is a website which allows us to view the status of the Lumino Network. Each node can decide to be present or not in this explorer.
+RIF Lumino Explorer is a website which allows ysers to view the status of the Lumino Network. Each node can decide to be present or not in this explorer.
```
Comment (mauricioirace) [inline rif/lumino/explorer/index.md]:
```suggestion
RIF Lumino Explorer is a website which allows users to view the status of the Lumino Network. Each node can decide to be present or not in this explorer.
```

File: rif/lumino/explorer/index.md [hunk 2]
```diff
@@ -1,42 +1,40 @@
...
 
-- Node graph: At the center of the screen we can see all the nodes which opted to be included in the explorer and how they connect between each other
-- Tokens dropdown: Above the graph, there is a dropdown which allows us to select which token we want to display information for.
-- Search box: allow to search by:
-  - Node address
-  - Channel id
-  - Token
-  - Ot the left, we have a summary of the network
-  - At the top right we have 2 buttons to go to:
+- **Node graph**: At the center of the screen we can see all the nodes which opted to be included in the explorer and how they connect toeach other.
```
Comment (mauricioirace) [inline rif/lumino/explorer/index.md]:
```suggestion
- **Node graph**: At the center of the screen we can see all the nodes which opted to be included in the explorer and how they connect to each other.
```

File: rif/lumino/explorer/index.md [hunk 3]
```diff
@@ -1,42 +1,40 @@
...
-
-Summary Widget
+## Summary Widget
 
 At the left of the dashboard we will see a summary widget with some general information about the network and the token selected in the dropdown.
 
 <img src="/assets/img/lumino/widget.png" alt=""/>
 
-In the Dashboard (Screenshots above) we got some statistics from the Mainnet.
+In the Dashboard we got can see statistics corresponding to Mainnet.
```
Comment (mauricioirace) [inline rif/lumino/explorer/index.md]:
```suggestion
In the Dashboard we can see statistics corresponding to Mainnet.
```

File: rif/lumino/light-client/use/index.md [hunk 4]
```diff
@@ -27,25 +28,22 @@ For more information, please take a look at the Readme indications at [https://g
 
     To start using the SDK, you must first configure the Lumino object. This object is our main interface to interact with the SDK. Lumino must be initialized before being used, invoking the init function:
 
-    ```
+    ```javascript
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md:31]: Does this work? I remember that for some reason, adding an specific language wasn't working in this markdown implementation.

File: rif/lumino/light-client/use/index.md [hunk 5]
```diff
@@ -65,13 +63,13 @@ For more information, please take a look at the Readme indications at [https://g
...
-    NOTE about the SigningHandler: This method requires the PrivateKey since we instantiate everything with web3 and ethers.js to sign Smart Contract transactions and Off Chain Messages, it is not compulsory to use it.
+    **NOTE** about the `signingHandler`: This method requires the `PrivateKey` since we instantiate everything with web3 and ethers.js to sign Smart Contract transactions and Off Chain Messages, it is not compulsory to use it.
 
     We advise only using it when testing in a secure environment, but in any case you may provide your own signing functions instead of using the handler.
 
-    The idea is passing an object with 2 functions instead of the signingHandler to Lumino.init(), these functions receive 1 parameter each
+    The idea is passing an object with 2 functions instead of the `signingHandler` to `Lumino.init()`, these functions receive 1 parameter each
 
-    ```
+    ```javascript
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md:72]: same question as above

File: rif/lumino/light-client/use/index.md [hunk 6]
```diff
@@ -84,48 +82,48 @@ For more information, please take a look at the Readme indications at [https://g
...
-3. **Make the onboarding**
+3. **Perform the onboarding**
 
-    In order to start using the Lumino light client, you need to make an onboarding into a Lumino HUB node. 
+    In order to start using the Lumino light client, you need to perform an onboarding into a Lumino HUB node. 
 
     First, define an async function that invokes the onboarding primitive:
 
-    ```
+    ```javascript
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md:91]: same question as above

File: rif/lumino/light-client/use/index.md [hunk 7]
```diff
@@ -84,48 +82,48 @@ For more information, please take a look at the Readme indications at [https://g
...
     };
     ```
 
-    Then you can use this function on a button action, for example. After that, your client is able to start interacting with the Lumino HUB
+    Then you can use this function on a button action, for example. After that, your client will be able to start interacting with the Lumino HUB.
 
 4. **Open a channel**
 
-    ```
+    ```javascript
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md:101]: same question as above

File: rif/lumino/light-client/use/index.md [hunk 8]
```diff
@@ -84,48 +82,48 @@ For more information, please take a look at the Readme indications at [https://g
...
+      Partner: "0x1234...",
       settleTimeout: 500,
-      tokenAddress: “0x1234...”,
+      tokenAddress: "0x1234...",
     };
     await Lumino.get().actions.openChannel(params);
     ```
 
-    The open channel function will fire and when the process is successful, a callback will be fired if specified (see the readme on Callbacks for more information on this topic). A channel structure will be added to the SDK state machine.
+    The open channel function will fire and when the process is successful, a callback will be fired if specified (see the readme on Callbacks down belo for more information on this topic). A channel structure will be added to the SDK state machine.
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md]:
```suggestion
    The open channel function will fire and when the process is successful, a callback will be fired if specified (see the readme on Callbacks down below for more information on this topic). A channel structure will be added to the SDK state machine.
```

File: rif/lumino/light-client/use/index.md [hunk 9]
```diff
@@ -84,48 +82,48 @@ For more information, please take a look at the Readme indications at [https://g
...
 
     You just need to specify the partner node (a lumino node) address and the token in which you want to create the channel. 
 
 5. **Deposit**
 
-    After a channel was created, you need to make an on-chain deposit in order to start making payments
+    After a channel was created, you need to make an on-chain deposit in order to start making payments.
 
-    ```
+    ```javascript
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md:118]: same question as above

File: rif/lumino/light-client/use/index.md [hunk 10]
```diff
@@ -135,12 +133,12 @@ For more information, please take a look at the Readme indications at [https://g
 
     Now that the channel has funds, you can make a payment to your partner. 
 
-    ```
+    ```javascript
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md:136]: same question as above

File: rif/lumino/light-client/use/index.md [hunk 11]
```diff
@@ -149,54 +147,55 @@ For more information, please take a look at the Readme indications at [https://g
...
 
 8. **Configuring callbacks**
 
-    All the operations previously described perform operations on background, but you may want to display success or error messages into your dapp after the operation finalises. For that, there are different callbacks.
+    All the operations previously described perform operations on background, but you may want to display success or error messages into your dApp after the operation finalises. For that, there are different callbacks.
 
     Callbacks are set on the Lumino instance like this:
 
-    ```
+    ```javascript
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md:163]: same question as above

File: rif/lumino/light-client/use/index.md [hunk 12]
```diff
@@ -149,54 +147,55 @@ For more information, please take a look at the Readme indications at [https://g
...
     * When the payment is completed: A callback will also be fired
 
-
-    Callbacks are explained on the next section
+    Callbacks are explained on the next section.
 
 8. **Configuring callbacks**
 
-    All the operations previously described perform operations on background, but you may want to display success or error messages into your dapp after the operation finalises. For that, there are different callbacks.
+    All the operations previously described perform operations on background, but you may want to display success or error messages into your dApp after the operation finalises. For that, there are different callbacks.
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md]:
```suggestion
    All the operations previously described perform operations on background, but you may want to display success or error messages into your dApp after the operation finishes. For that, there are different callbacks.
```

File: rif/lumino/light-client/use/index.md [hunk 13]
```diff
@@ -149,54 +147,55 @@ For more information, please take a look at the Readme indications at [https://g
...
 
-    The callbacks.set method sets a function that may or may not receive the data regarding the event, the next table illustrates the callbacks, when they are fired and the data they provide (Which is always a single object or nothing)
+    The `callbacks.set` method sets a function that may or may not receive the data regarding the event, the next table illustrates the callbacks, when they are fired and the data they provide (which is always a single object or nothing).
 
-    We recommend creating a callbacks.js file that encapsulates all the callbacks. For example:
+    We recommend creating a `callbacks.js` file that encapsulates all the callbacks. For example:
 
-    ```
-    // // Set callbacks
+    ```javascript
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md:171]: same question as above

File: rif/lumino/light-client/use/index.md [hunk 14]
```diff
@@ -208,12 +207,14 @@ For more information, please take a look at the Readme indications at [https://g
...
-    * [https://github.com/rsksmart/rif-notifier](https://github.com/rsksmart/rif-notifier)
+    * [Lumino Light Client SDK: RIF Notifier](https://github.com/rsksmart/lumino-light-client-sdk#rif-notifier)
+    * [Rif-Notifier](https://github.com/rsksmart/rif-notifier)
+
+    &nbsp;
 
     Having a RIF Notifier up and running, the dApp must register to that notifier instance and subscribe to open channel events. 
 
-    ```
+    ```javascript
```
Comment (mauricioirace) [inline rif/lumino/light-client/use/index.md:217]: same question as above

**Review by mauricioirace — COMMENTED** [review]
Just a few typos, and a question regarding setting the language of a snippet

**Review by mauricioirace — APPROVED** [review]
LGTM!

**Comment by mortelli** [comment]
> try out the explorer API guide
try out the explorer WEB guide

@mauricioirace has taken care of these 2 (thanks!)

and we've decided this one:
> try out the lumino SDK js guide

is out of scope for now.

the rest of the changes are being uploaded directly to this repo

**Pull request merged**